### PR TITLE
MonthDayView draw dot for each event instead of just one dot.

### PR DESCRIPTION
**Pull request by themangosteen**
This is an attempt to implement #38. Please note that I have not done any programming in a long time and study biology now, so I hope this will still be helpful.. Btw I am so happy that you folks are continuing Tibi's project in Fossify. Thank you! 

There might be issues with scaling on different resolutions. And the drawing of the little plus sign maybe is better with a ShapeDrawable instead of text. I am not sure.

### CODE REVIEW

**Comment by themangosteen** [comment]
There might be issues with scaling on different resolutions. And the drawing of the little plus sign maybe is better with a ShapeDrawable instead of text. I am not sure.

File: app/src/main/kotlin/org/fossify/calendar/views/MonthView.kt [hunk 1]
```diff
@@ -190,16 +198,37 @@ class MonthView(context: Context, attrs: AttributeSet, defStyle: Int) : View(con
...
+                        for ((index, event) in dayEventsSorted.withIndex()) {
+                            indexInRow = index % dotsPerRow
+                            xDot = xPosCenter + stepSize * (indexInRow - (min(eventCount, dotsPerRow)) / 2)
+                            if (eventCount % 2 == 0) { // center even number of dots
+                                xDot += stepSize / 2
+                            }
+                            if (index > 0 && indexInRow == 0) { // next row of dots
+                                yDot += stepSize
+                            }
+                            if (index >= dotsPerRow * 2 - 1) { // draw + if too many events
```
Comment (naveensingh) [inline app/src/main/kotlin/org/fossify/calendar/views/MonthView.kt:224]: About line 224, the `+` symbol shouldn't be drawn when `eventCount` matches `EVENT_DOT_COLUMN_COUNT` (i.e. dotsPerRow). It should only show up when `eventCount` exceeds the dots per row count.

```kotlin 
 // draw + if too many events
if (index >= dotsPerRow * EVENT_DOT_COLUMN_COUNT - 1 && eventCount != index - 1) {}
```

**Comment by naveensingh** [comment]
> Please note that I have not done any programming in a long time and study biology now

Just so you know my suggestions above are optional, I'll apply the suggested changes if you can't (or if you are not feeling up to it)

**Comment by themangosteen** [comment]
If you can do it, I would highly appreciate it. I only tried to do a rough sketch really, and am not familiar with current best practises in these software environments. Thank you!!

**Comment by naveensingh** [comment]
Seems to work fine, thanks!

**Pull request merged**